templates/definition/charger/lg-therma.yaml

```
@@ -0,0 +1,69 @@
+template: lg-therma
+products:
+  - brand: LG
+    description:
+      generic: Therma V R290 Monobloc (SG Ready)
+group: heating
+requirements:
+  description:
+    de: |
+      Modbus Verbindung zur LG Therma V Wärmepumpe.
+      Funktioniert über SG Ready Energiezustände für intelligente Steuerung.
+      Inneneinheit/Fernbedienung in den erweiterten Einstellungen > Konnektivität > Energiezustand > Signaltyp > auf "Modbus" einstellen.
+      Da die interne Energiemessung zu hohe Werte anzeigt ist eine externe Energiemessung empfohlen.
+      Werkseinstellung Modbus Addresse(HEX) = 21 (33)
```

Add further explanation.
```suggestion
      Werkseinstellung Modbus Addresse(HEX) = 21 (33).
      Modbus Register sind im Installationshandbuch zu finden (Adresse jeweils ohne die führende Ziffer und -1).
      Informationen zu den Energiezuständen finden sich auch im Installationshandbuch.
```